.ci/setup.sh

```diff
@@ -41,6 +41,9 @@ pushd "${runtime_repo_dir}"
 sed -i 's/cc-agent.target/clear-containers.target/' create.go
 popd
 
+# Install libudev-dev required for go-udev vendor dependency
+sudo apt-get install libudev-dev
```

Comment:
This commit can be removed since this is installed from the `tests` repo. https://github.com/clearcontainers/tests/pull/337 just got merged.